qiskit/ml/circuit/library/__init__.py

```diff
@@ -0,0 +1,19 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The circuit library for Machine Learning applications."""
+
+from .raw_feature_vector import RawFeatureVector
+
+__all__ = [
```

Comment:
Since this is adding a new component and module we need to have it in the docs. Can you add a `circuit` entry to the ml init file for this new submodule. This then needs a docstring entry here for RawFeatureVector class - maybe the table form linking to the class under some library heading. Then we should have a qiskit.ml.circuit.rst stub file in the apidocs like the others there.

Reply:
I tried, let's see if it works 😉